gradle/distributionTesting.gradle

```diff
@@ -182,7 +182,7 @@ def removeCachedScripts(File cachesDir) {
     if (cachesDir.directory) {
         for (File cacheDir : cachesDir.listFiles()) {
             if(cacheDir.directory) {
-                ["scripts", "scripts-remapped"].each {
+                ["scripts", "scripts-remapped", "gradle-kotlin-dsl", "gradle-kotlin-dsl-accessors"].each {
```

Comment:
:o: How about changing `kotlin-dsl` to use a single directory named `kotlin-dsl` and simplify the change on this side?

Reply:
https://github.com/gradle/gradle/blob/609f28d99e02354e3c69c6bc3a19e7284eccc13b/subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/http/LegacyArtifactLookupResolveIntegrationTest.groovy

Reply:
@bamboo this would require sharing the compilation and accessors-generation caches on the `kotlin-dsl` side, we could look into it yes, but I think we should merge this like it is to get the build correctness right first.

Reply:
@eskatos makes sense, could you create a followup issue in the `kotlin-dsl` board then?

Reply:
Done https://github.com/gradle/kotlin-dsl/issues/623